api_tests/requests/mixins.py

```diff
@@ -107,18 +107,17 @@ def pre_mod_preprint(self, admin, write_contrib, pre_mod_provider):
         pre = PreprintFactory(
             creator=admin,
             provider=pre_mod_provider,
-            is_published=False,
-            machine_state='pending'
+            is_published=True,
+            machine_state='accepted',
         )
         pre.ever_public = True
+        pre.is_public = True
```

Comment:
Curious why the new and previous set-up would lead to testing result difference:
* New: set published and accepted
* Previous: set pending and unpublished and then make it published

Maybe there is a bug in our factory (or even code) that needs to fix?

Reply:
removed this